### ISSUE TITLE
Turns message to bold and changes font

### ISSUE THREAD

**Issue by tmjvasiga** [opto/Text-Insert#4]
When hitting reply in TB 115.5.1 on Windows 11, the message is bolded and the font changed to Arial, rather than the normal default reply settings.

**Comment by opto** [comment]
yes, I noted that bug today as well. It might help to switch off the spellcheck or set it to a language you are not using.

**Comment by tmjvasiga** [comment]
That didn't seem to fix it for me.  The spellcheck is off and set to Dutch, which I won't be using.

**Comment by opto** [comment]
yes, I found it, the borrowed styling for the selectbox did set the font to bold.

Also, I noticed that the select box for alternatives is shown at the very end of the mail - bad, if it is a reply with a citation. I had tested only creating my emails.

I will upload a new version tomorrow

**Comment by cheonman** [comment]
In Thunderbird 115.5.1 (Windows 10) with v1.5.3 I get this issue for new messages as well as replies. The add-on was updated today (Dec. 5), so I'm not sure if this already includes your fixes.

**Comment by opto** [comment]
I just uploaded1.5.6 which solves this, sorry for the bug

**Comment by Thomas-D-C** [comment]
Hi :-)

What's the difference between the resolved label and a closed issue ?
